test/language/expressions/arrow-function/arrow/apply_2.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: invoke arrow function using apply - 'this' is lexically bound
+es6id: 14.2
+---*/
+
+var b = {
+    method: () => this
+ }
+
+var a = b.method;
+
+assert.sameValue(a.apply(null), this, "Invocation of arrow function failed using fn.apply(null).");
+
+assert.sameValue(a.apply(undefined), this, "Invocation of arrow function failed using fn.apply(undefined).");
+
+assert.sameValue(a.apply(this), this, "Invocation of arrow function failed using fn.apply(this).");
+
+assert.sameValue(a.apply({}), this, "Invocation of arrow function failed using fn.apply({}).");

```

Comment:
Duplicates

- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/lexical-this.js